server/services/utils/getServicesForUser.test.ts

```diff
@@ -29,6 +29,7 @@ jest.mock('../../config', () => ({
     sendLegalMail: { url: 'url' },
     welcomePeopleIntoPrison: { url: 'url', enabledPrisons: 'LEI,LIV' },
     mercurySubmit: { url: 'url' },
+    imsSubmit: { url: 'url' },
```

Comment:
Can we change this to manageIntelligence

Reply:
changed to manageIntelligence